src/index.js

```diff
@@ -1,43 +1,192 @@
-      <label type="text" id="${task.index}" for="${task.description}">${task.description}</label>
+    <input type="checkbox" id="${index} ${task.completed ? 'checked' : ''}" class="check">
+    <label type="text" class="Label" id="pTask${index}">${task.description}</label>
+    <input type="text" class="UpdateLabel" id="updt${index}"/>
+    </div>  
+    <div class="hide2" id="ellipse${index}">
+    <i class="fa-solid fa-ellipsis-vertical" ></i>
+    </div>
+    <div class="hide1" id="delete${index}">
+    <i id="idBtn${index}" class="fa-solid fa-trash-can" ></i>
     </div>
-    <i class="fa-solid fa-ellipsis-vertical"></i>
     </li>
     `;
   });
   toDoList.innerHTML = display;
+
+  showTrash();
+};
+
+const saveTask = ({ index, description, completed = false }) => {
+  storedTasks = [];
+
+  if (localStorage.getItem('tasks') === null) {
+    storedTasks = [];
+  } else {
+    storedTasks = JSON.parse(localStorage.getItem('tasks'));
+  }
+
+  storedTasks.push({ index, description, completed });
+  localStorage.setItem('tasks', JSON.stringify(storedTasks));
+  getTask();
+};
+
+const saveEditTask = (task) => {
+  const taskItem = storedTasks[task];
+  taskItem.description = addTaskInput.value;
+  localStorage.setItem('tasks', JSON.stringify(storedTasks));
+};
+
+addTask.addEventListener('click', (addItem) => {
+  addItem.preventDefault();
+  if (!addTaskInput.value) return;
+
+  if (editTaskItem != null) {
+    saveEditTask(editTaskItem);
+    editTaskItem = null;
+  } else {
+    saveTask({ index: storedTasks.length, description: addTaskInput.value, completed: false });
+  }
+  addTaskInput.value = '';
+  window.location.reload();
+});
+
+document.addEventListener('keydown', (press) => {
+  if (press.key === 'Enter' && addTaskInput.value) {
+    addTask.click();
+  }
+});
+
+const SteerChecked = () => {
+  const StoreCheck = JSON.parse(localStorage.getItem('tasks'));
+  const checkBoxs = document.querySelectorAll('.check');
+  StoreCheck.forEach((a, i) => {
+    if (a.completed === true) {
+      checkBoxs[i].checked = true;
+      document.getElementById(`ptask${i}`).style.textDecoration = 'line-through rgb(68, 68, 68)';
+    }
+  });
+};
+
+const removeChecked = () => {
+  const buttonRemove = document.getElementById('button-clear');
+  const checkBoxs = document.querySelectorAll('.check');
+  const TasksR = JSON.parse(localStorage.getItem('tasks'));
+  checkBoxs.forEach((check, i) => {
+    check.addEventListener('click', () => {
+      if (TasksR[i].completed === true) {
+        TasksR[i].completed = false;
+        checkBoxs[i].checked = false;
+        document.getElementById(`pTask${i}`).style.textDecoration = 'none';
+        localStorage.setItem('tasks', JSON.stringify(TasksR));
+      } else {
+        TasksR[i].completed = true;
+        checkBoxs[i].checked = true;
+        document.getElementById(`ptask${i}`).style.textDecoration = 'line-through rgb(68, 68, 68)';
```

Comment:
- [ ] Kindly do the same here
- [ ] Make sure to fix the typo `ptask`, the element you are trying to target with `document.getElementById('ptask${i}')` has an id of `pTask*`